html/pfappserver/lib/pfappserver/Form/Config/Authentication/Source/AdminProxy.pm

```diff
@@ -0,0 +1,66 @@
+package pfappserver::Form::Config::Authentication::Source::AdminProxy;
+
+=head1 NAME
+
+pfappserver::Form::Config::Authentication::Source::AdminProxy - Form for the AdminProxySource
+
+=cut
+
+=head1 DESCRIPTION
+
+pfappserver::Form::Config::Authentication::Source::AdminProxy
+
+=cut
+
+use strict;
+use warnings;
+use HTML::FormHandler::Moose;
+extends 'pfappserver::Form::Config::Authentication::Source';
+
+has_block definition => (
+    render_list => [qw(proxy_addresses user_header group_header)],
+);
+
+has_field 'proxy_addresses' => (
```

Comment:
Can there be multiple addresses ? If so, this needs a validator as well as help text on how to separate the addresses